packages/webapp/src/containers/Task/index.jsx

```diff
@@ -27,7 +27,7 @@ import {
   userFarmSelector,
 } from '../userFarmSlice';
 import { resetAndUnLockFormData } from '../hooks/useHookFormPersist/hookFormPersistSlice';
-import { getCropsAndManagementPlans } from '../saga';
+import { getManagementPlansAndTasks } from '../saga';
```

Comment:
The changes in this file are directly undoing the changes from this PR:

- https://github.com/LiteFarmOrg/LiteFarm/pull/2829